src/CMakeLists.txt

```diff
@@ -167,6 +168,7 @@ target_link_libraries(migraphx PRIVATE -lstdc++fs)
 endif()
 
 target_include_directories(migraphx SYSTEM PUBLIC $<BUILD_INTERFACE:${HALF_INCLUDE_DIR}>)
+include_directories(${CMAKE_CURRENT_BINARY_DIR})
```

Comment:
The include should be added to the target: `target_include_directories(migraphx PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/include>)`.

Reply:
Fixed